bears/js/JSStandardBear.py

```diff
@@ -14,15 +15,65 @@ class JSStandardBear:
     No .eslintrc, .jshintrc, or .jscsrc files to manage.
     It just works.
     """
-
     LANGUAGES = {'JavaScript', 'JSX'}
-    REQUIREMENTS = {NpmRequirement('standard', '10')}
+    REQUIREMENTS = {NpmRequirement('standard', '10'),
+                    NpmRequirement('babel-eslint', '8.0'),
+                    NpmRequirement('eslint-plugin-flowtype', '2.41.0'),
+                    NpmRequirement('typescript-eslint-parser', '12.0.0'),
+                    NpmRequirement('eslint-plugin-typescript', '0.8.1'),
+                    NpmRequirement('eslint-plugin-markdown', '1.0.0-beta.6'),
+                    NpmRequirement('eslint-plugin-html', '3.2.2'), }
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Syntax'}
     CAN_FIX = {'Formatting'}
     SEE_MORE = 'https://standardjs.com/rules.html'
 
-    def create_arguments(self, filename, file, config_file):
-        return (filename, '--verbose')
+    def create_arguments(self, filename, file, config_file,
+                         globalVars: typed_list(str)=(),
+                         typescript: bool=False,
+                         flow: bool=False,
+                         env: str='',
+                         markdown: bool=False,
```

Comment:
nope; this should take a `language: Language`, accepting HTML/Markdown, and add the necessary parameters behind the scenes.

TypeScript/flow should also be defined languages, IMO, and should be handled the same way, but they are at least variants of JavaScript so maybe others have an alternative opinion about allowing those to be parameters. 

You may need to define some of those languages in coala core's coalang.

You might like to split this patch into two, one adding `env` and `globalVars`, since those are less complicated.

Reply:
okay yes that would be much cleaner rather than having all these options. Currently flow and markdown are not included in the coalang. Should I simply make a PR to add these two languages first and then finish the work here? @jayvdb